### PR TITLE
add db:test:prepare

### PR DESCRIPTION
**Pull request by mmoll**
I ran into this some time ago.

### CODE REVIEW

**Comment by dLobatog** [comment]
`rake test` runs `rake db:test:prepare` automatically. Running tests manually with `ruby -Itest test/functional/your_test.rb` does not, so you'd need to run `rake db:test:prepare` before. Try to convey that explanation in here :smile:

**Comment by mmoll** [comment]
like this? :laughing:

File: contribute.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ You can also exclude these features by using `bundle install --without libvirt p
 1. Copy `config/database.yml.example` to `config/database.yml`
 2. Create your database: `rake db:migrate`
 3. Run all the tests: `rake test`
-4. Or a single test: `ruby -Itest test/functional/your_test.rb`
+4. Or a single test: `ruby -Itest test/functional/your_test.rb` (if you did't run all tests before, you need to prepare the test environment with `rake db:test:prepare` the first time).
```
Comment (dLobatog) [inline contribute.md]: did't -> didn't

Comment (mmoll) [inline contribute.md]: sorry, fixed :bowtie:

**Comment by stbenjam** [comment]
Thanks @mmoll!

**Pull request merged**